### PR TITLE
Post-Processing API

### PR DESCRIPTION
**Pull request by jleandroperez**
#### Details:
In this PR we're exposing a new public method, that grants access to the internal `UINavigationController`. This is required to wire a post-processing step, such as cropping.

Needs Review: @SergioEstevao 
Thanks in advance sir!

### CODE REVIEW

File: Pod/Classes/WPMediaPickerViewController.m [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #import "WPPHAssetDataSource.h"
 
 @interface WPMediaPickerViewController () <UINavigationControllerDelegate>
-
+@property (nonatomic, strong) UINavigationController *internalNavigationController;
```
Comment (SergioEstevao) [inline Pod/Classes/WPMediaPickerViewController.m]: I think this could just be called navigationController no need for internal.

**Comment by SergioEstevao** [comment]
@jleandroperez don't you want to make the change for not dismissing the view controller when doing a single selection?

Do you mind to add a sample of this functionality on the example app? It can just a dummy view controller that show the image in order to be tested there.

File: Pod/Classes/WPMediaPickerViewController.m (hunk 1 above)
Comment (jleandroperez) [inline Pod/Classes/WPMediaPickerViewController.m]: @SergioEstevao `navigationController` would actually collide with `UIViewController`'s **navigationController** property.

**Comment by jleandroperez** [comment]
@SergioEstevao ready for another look sir!. Added `showDoneButton` property (suggestions for a proper name would be more than welcome!)

**Comment by SergioEstevao** [comment]
:shipit:

**Comment by jleandroperez** [comment]
Yay, thanks for the review @SergioEstevao!!

**Pull request merged**